src/panel/poi/PoiPanel.jsx

```diff
@@ -157,20 +155,6 @@ export default class PoiPanel extends React.Component {
     window.app.navigateTo('/favs');
   }
 
-  showDetails = () => {
-    const poi = this.getBestPoi();
-    Telemetry.add(Telemetry.POI_SEE_MORE, null, null,
```

Comment:
The corresponding constant (`poi_see_more`) in `local_modules/telemetry/index.js` can be removed.